homeassistant/components/asuswrt/__init__.py

```diff
@@ -66,6 +72,8 @@
         conf.get("ssh_key", conf.get("pub_key", "")),
         conf.get(CONF_MODE),
         conf.get(CONF_REQUIRE_IP),
+        conf.get(CONF_INTERFACE),
+        conf.get(CONF_DNSMASQ),
```

Comment:
Please use `dict.get(key)` only for optional keys _without_ default values and `dict[key]` for required keys and optional keys _with_ default values.

Reply:
Hmm,ok, I was looking at SSH port which also is optional with default value and set as dict.get(key). So should I update that one as well to use dict[key] instead for cleanness?

Reply:
Yes please 👍

Reply:
done